react-exercise/src/usersClientService.ts

```diff
@@ -1,35 +1,34 @@
-import { fetchUsers, createUser, updateUser as updateUserOnServer, deleteUser } from './usersDBClient';
-import { addUser, setUsers, updateUser as updateUserOnStore, removeUser } from './stores/usersStore';
+import { fetchUsers as fetchUsersOnServer, createUser, updateUser as updateUserOnServer, deleteUser } from './usersDBClient';
+import { addUser as addUserOnStore, setUsers as setUsersOnStore, updateUser as updateUserOnStore, removeUser } from './stores/usersStore';
 import { User } from './types/user';
 
-// Sync all users from server to Zustand store
-export const fetchUsersSync = async (
```

Comment:
Note that the same remark regarding the "sync" name holds for the other methods here as well, 
not super critical to change, just take it into consideration in the future

Reply:
Yes I agreem this is why I dropped in the round commit the sync from all the methods